.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Kpt Slack Channel
+    url: https://kubernetes.slack.com/app_redirect?channel=kpt
+    about: Discuss Kpt Topics with the Kpt Team on Slack
+  - name: Kpt Guides
```

Comment:
We also have an email group at https://googlecontainertools.github.io/kpt/contact/